internal/idle/idle.go

```diff
@@ -266,6 +267,7 @@ func (m *Manager) isClosed() bool {
 	return atomic.LoadInt32(&m.closed) == 1
 }
 
+// Close stops the timer associated with the Manager, if it exists.
```

Comment:
Should we say something like?

```suggestion
// Close prevents calls to enter/exit idle immediately.  This call also
// ensures we are not entering/exiting idle mode. 
```

Stoping the timer seems like a side effect?